recipe/meta.yaml

```diff
@@ -12,15 +12,9 @@ source:
     url: https://github.com/google/protobuf/archive/{{ version_str }}/{{ name }}-{{ version_str }}.tar.gz
     sha256: {{ sha256 }}
     patches:
-      # Add a patch for non-C++11 support.
-      - field_mask_util.cc.patch
-      # Adds a patch to get rid of some constants also included in a class.
-      - time_util_const_dedup.patch
 
 build:
     number: 0
-    # skip the windows python 2.7 and 3.4 build
-    # see https://github.com/google/protobuf/issues/1958
```

Comment:
LOL. Sorry for leaving this errant comment in there. Thanks for cleaning it out.